ml-agents/mlagents/trainers/models.py

```diff
@@ -112,7 +112,11 @@ def create_learning_rate(
             learning_rate = tf.Variable(lr)
         elif lr_schedule == LearningRateSchedule.LINEAR:
             learning_rate = tf.train.polynomial_decay(
-                lr, global_step, max_step, 1e-10, power=1.0
+                np.float32(lr),
```

Comment:
I should change this back - I was originally trying to prevent tensorflow from creating float64 arrays internally, but gave up on that fight.